src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -227,21 +235,24 @@ describe('CdkVirtualScrollViewport', () => {
     it('should set content offset to top of content', fakeAsync(() => {
       finishInit(fixture);
       viewport.setRenderedContentOffset(10, 'to-start');
+      fixture.changeDetectorRef.markForCheck();
       fixture.detectChanges();
       flush();
 
       expect(viewport.getOffsetToRenderedContentStart()).toBe(10);
     }));
 
-    it('should set content offset to bottom of content', fakeAsync(() => {
+    // TODO: verify
+    fit('should set content offset to bottom of content', fakeAsync(async () => {
+      fixture.autoDetectChanges();
       finishInit(fixture);
       const contentSize = viewport.measureRenderedContentSize();
 
       expect(contentSize).toBeGreaterThan(0);
 
       viewport.setRenderedContentOffset(contentSize + 10, 'to-end');
-      fixture.detectChanges();
       flush();
+      await fixture.whenStable();
```

Comment:
This change makes sense because `flush` won't cause CD to happen like it would with zones. The zoneless scheduler is run in the root zone so `flush` doesn't flush it.